.drone.yml

```diff
@@ -47,5 +46,35 @@ steps:
         include:
           - master
       event:
+        include:
+          - push
+          - tag
         exclude:
           - pull_request
+
+  - name: gen-manifests
+    image: golang:latest
+    commands:
+      - wget -O /usr/local/bin/kustomize https://github.com/kubernetes-sigs/kustomize/releases/download/v1.0.11/kustomize_1.0.11_linux_amd64
+      - chmod +x /usr/local/bin/kustomize
+      - make manifests
+      - cat provider-components.yaml
+    environment:
+      IMG: "autonomy/cluster-api-provider-talos:${DRONE_TAG}"
```

Comment:
Not for this PR but we should eventually align this with how we are doing tags in all our other repos.

Reply:
Actually, `IMG` seems to only be used in the `Makefile`. This should be a quick win. Can we align this with our other repos?

```
TAG = $(shell gitmeta image tag)
```